src/applications/ask-va/sass/ask-va-too.scss

```diff
@@ -141,7 +141,7 @@ $facility-locator-shadow: rgba(0, 0, 0, 0.5);
     width: 18px;
     text-align: center;
     margin-right: 8px;
-    color: var(--vads-color-link);
+    color: $color-link-default;
```

Comment:
@ediiotero Hi Edwin! I'm know it's been a while since this was PR was merged but do you remember why the CSS variable was changed back to the Sass variables in this file? The design system will probably be deprecated these Sass variables in the near future but we want to understand if there are some limitations in doing that.